tests/units/units.cpp

```diff
@@ -1725,27 +1862,31 @@ TEST(Units, isEquivalentBaseUnitNotInModel)
     u2->setName("u2");
     u2->addUnit(libcellml::Units::StandardUnit::RADIAN, 0, 1.0, 1.0);
     u2->addUnit("u", 0, 1.0, 1.0);
+    // test should fail because base unit "u" is not present in u1, regardless of whether "apples" exists or not
 
     model->addUnits(u1);
     model->addUnits(u2);
+    // model->linkUnits();
+    // Units "u" not in model.  Units "apples" doesn't exist.  Units are not linked.
+    // Previously returned false because of mismatched number of child units.
```

Comment:
Previously? What does this mean?

Reply:
Previously = earlier behaviour of the equivalence testing functions.  I'll remove the comment, no longer relevant.